sql/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM skiplabs/skdb-base as skdb
+FROM skiplabs/skdb-base AS skdb
```

Comment:
Tangential: We should probably introduce a base image `skiplabs/skiplang-base` in `skiplang/Dockerfile`, which contains only an installed skiplang toolchain.